dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflow.java

```diff
@@ -17,6 +18,7 @@
     @Override
     public boolean execute(ExecutionContext context) {
         SimpleRuntimeService srs = (SimpleRuntimeService) context.getRuntimeService();
+        BaseFile abc = getSourceFile(context, "/tmp/test.txt");
```

Comment:
I am not a fan of polluting /tmp. Roddy already puts so many files there that have to be cleaned up manually. We should consider making an issue for cleaning up /tmp after tests.

Here the file should be created in a dedicated subdirectory for /tmp.

BTW: `abc`?

Reply:
This is really only a test workflow which is currently not used in any kind of automatic test.
The file also does not exist.